composer.json

```diff
@@ -18,6 +18,7 @@
     ],
     "require": {
         "php": "^5.3 || ^7.0",
+        "guzzlehttp/guzzle": "^6.1",
```

Comment:
Not sure if we really need this dependency for the basic setup. IMHO the media bundle should work without the amazon part, the guzzle dependency